src/app/profiles/profile-constants.ts

```diff
@@ -0,0 +1,48 @@
+export const constants = {
+  'services': {
```

Comment:
Why are you using single quotes around these keys?

Reply:
Initially, it was a JSON file but then I turned into a constants file.  Then linter complained about the double-quotes so I replaced all with single quotes.  Just an explanation, not a good reason.

Reply:
Fixed